exp/cmd/migrate_managed_cluster/migrate_managed_cluster.go

```diff
@@ -0,0 +1,131 @@
+package main
```

Comment:
no idea how to package this or where to put it. but I thought it's a nice touch!